website/siteConfig.js

```diff
@@ -29,7 +29,7 @@ const users = [
 
 const siteConfig = {
   title: "Apache APISIX™",
-  tagline: "The Cloud-Native API Gateway",
+  tagline: "A dynamic, real-time, high-performance API gateway, based on the Nginx library and etcd.",
```

Comment:
```
Apache APISIX is a dynamic, real-time, high-performance Cloud-Native API gateway, based on the Nginx library and etcd.

Apache APISIX software provides rich traffic management features such as load balancing, dynamic upstream, canary release, circuit breaking, authentication, observability, and more.
```

Reply:
And we need add this desc in download page too.

Reply:
![image](https://user-images.githubusercontent.com/2106987/93170581-133b6b80-f75a-11ea-8b29-148a796233b8.png)

Too long to make it look better, but works.

Reply:
![image](https://user-images.githubusercontent.com/2106987/93171082-01a69380-f75b-11ea-96bd-4855d483d295.png)